core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/proto/javadsl/ProtoInlet.scala

```diff
@@ -30,4 +36,9 @@ final case class ProtoInlet[T <: GeneratedMessageV3](name: String, clazz: Class[
   def schemaDefinition = ProtoUtil.createSchemaDefinition(descriptor)
 
   def withUniqueGroupId: ProtoInlet[T] = if (hasUniqueGroupId) this else copy(hasUniqueGroupId = true)
+  override def withErrorHandler(handler: (Array[Byte], Try[T]) => Option[T]): CodecInlet[T] = {
+    errorHandler = handler
```

Comment:
need to copy in the error handler, do not use vars.

Reply:
Done